### DISCUSSION TITLE
Are errors from CustomEditorProvider.openCustomDocument cached?

### DISCUSSION THREAD

**Discussion by lppedd** [microsoft/vscode-discussions#2043] [Extension Development QnA]
It looks like if an `Error` is thrown from `CustomEditorProvider.openCustomDocument`, the state of the resource is cached somewhere.
That means trying to open the same resource `Uri` again won't work until VS Code is reloaded or reopened, as `openCustomDocument` won't ever be called again.

Is this a bug, or is it expected behavior?

**Comment by isidorn** [comment]
Sounds like you are facing a problem when using vscode. I suggest to open an issue here https://github.com/microsoft/vscode/issues
